controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -685,24 +697,18 @@ bool wfa_ca::get_send_1905_1_tlv_hex_list(std::list<tlv_hex_t> &tlv_hex_list,
                 return false;
             }
 
-            // Validate hex notation on value or list of values separated by space
+                // Validate hex notation on list of values separated by space
             auto values = string_utils::str_split(it->second, ' ');
-            for (auto value : values) {
+                for (const auto &value : values) {
                 if (!validate_hex_notation(value)) {
-                    err_string = "param name '" + lookup_str + "' has invalid hex notation value";
+                        err_string =
+                            "param name '" + lookup_str + "' has invalid hex notation value";
                     return false;
                 }
-
-                // Type and Length fields must have fixed size:
-
-                // Type
-                if (tlv_member_idx == 0 && value.size() != 4) {
-                    err_string = "type field is too large: " + value;
-                    return false;
-
-                    // Length
-                } else if (tlv_member_idx == 1 && value.size() != 6) {
-                    err_string = "length field is too large: " + value;
+                }
+            } else {
+                LOG(ERROR) << "Illegal tlv_member_idx value: " << int(tlv_member_idx);
+                err_string = err_internal;
                     return false;
```

Comment:
also wrong indentation

Reply:
I have no idea, but it was fixed in follow-up commit because I don't see it in my code.
This fix requires to rebase with edit, so I will fix it when I'll rebase.
Keeping it unresolved to remember...